### PR TITLE
fix: add experimentation for plus members

### PR DESCRIPTION
**Pull request by rebelchris**
Add experimentation exposure for plus members.

If user is plus it looks like this:

```
  "exp": {
    "f": "{}",
    "e": [],
    "a": {
      "show_error": 1,
      "plus": 1
    }
  },
```

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

File: src/routes/boot.ts [hunk 1]
```diff
@@ -530,7 +531,13 @@ const loggedInBoot = async ({
       notifications: { unreadNotificationsCount },
       squads,
       accessToken,
-      exp,
+      exp: {
+        ...exp,
+        a: {
+          ...exp.a,
+          ...(isPlus ? { plus: 1 } : undefined),
```
Comment (capJavert) [inline src/routes/boot.ts:538]:
```suggestion
          plus: isPlus ? 1 : undefined
```

its json so undefined will disappear

**Review by capJavert — APPROVED** [review]

File: src/routes/boot.ts [hunk 2]
```diff
@@ -476,6 +476,10 @@ const loggedInBoot = async ({
     const isTeamMember = exp?.a?.team === 1;
     const isPlus = isPlusMember(user.subscriptionFlags?.cycle);
 
+    if (isPlus) {
+      exp.a.plus = 1;
```
Comment (capJavert) [inline src/routes/boot.ts:480]: Is `a` object here, if its undefined it will break?

Comment (rebelchris) [reply src/routes/boot.ts:480]: Defaults to empty object I think:
https://github.com/dailydotdev/daily-api/blob/cfd63715cf57b59672611a93e514172711883153/src/routes/boot.ts#L362-L365

Comment (capJavert) [reply src/routes/boot.ts:480]: jep!

**Review by capJavert — APPROVED** [review]

**Review by idoshamun — APPROVED** [review]

**Pull request merged**